extensions/integration-tests/src/utils.ts

```diff
@@ -105,7 +105,7 @@ export async function createDB(dbName: string, ownerUri: string): Promise<void>
 			SELECT ERROR_MESSAGE() AS ErrorMessage;
 		END CATCH`;
 
-	let dbCreatedResult = await this.runQuery(query, ownerUri);
+	let dbCreatedResult = await runQuery(query, ownerUri);
```

Comment:
not inside a class, and "this" doesn't make sense here. removing it.

Reply:
functions can have a this context, I would verify with the owner first.

Reply:
just verified, removing it works fine. and also debugged it and you are right, this is valid and pointing the correct method. but I would still remove it since this is of type any, we won't get compilation error even if we use a method name doesn't exist.